packages/blockchain-wallet-v4-frontend/src/data/analytics/sagas.js

```diff
@@ -33,7 +33,7 @@ export default ({ api }) => {
       if (test(/cash/, text)) return yield call(api.logClick, 'bch')
       if (test(/stellar/, text)) return yield call(api.logClick, 'xlm')
       if (test(/(buy|sell)/, text)) return yield call(api.logClick, 'buysell')
-      if (test(/exchange/, text)) return yield call(api.logClick, 'exchange')
+      if (test(/swap/, text)) return yield call(api.logClick, 'swap')
```

Comment:
Can we leave the (api.logClick, 'exchange') and go over to swap when we add in the new service, which we can discuss offline?